language-plutus-core/test/Evaluation/Constant.hs

```diff
@@ -1,15 +1,14 @@
-module Evaluation.Constant.All
+module Evaluation.Constant
     ( test_constant
     ) where
 
 import           Evaluation.Constant.Success
-import           Evaluation.Constant.SuccessFailure
 
 import           Test.Tasty
 
+-- TODO: why don't we have any tests for 'EvaluationFailure'?
 test_constant :: TestTree
 test_constant =
     testGroup "constant"
         [ test_constantSuccess
-        , test_applyBuiltinNameSuccessFailure
```

Comment:
Those were useless tests, because they stopped testing failures after we'd removed sizes. Also deleted the entire file with these tests.